### PR TITLE
Added functionality to create a VariantCallingAnnotation

### PR DESCRIPTION
**Pull request by andrewmchen**

### CODE REVIEW

**Comment by fnothaft** [comment]
Jenkins, add to whitelist and test this please.

File: avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package org.bdgenomics.avocado.algorithms.math
```
Comment (fnothaft) [inline avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala]: Can you add the copyright header to this file?

Comment (andrewmchen) [inline avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala]: I remember you saying something about a linter/script to do this kind of functionality? Where can I find that?

File: avocado-core/src/main/scala/org/bdgenomics/avocado/genotyping/annotators/Annotator.scala [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            observations: Iterable[AlleleObservation],
+            variantQuality: Double,
+            likelihoods: Array[Double]): VariantCallingAnnotator = {
+    new VariantCallingAnnotator(variant, observations, variantQuality, likelihoods)
+  }
+}
+
+class VariantCallingAnnotator(variant: Variant, observations: Iterable[AlleleObservation], variantQuality: Double, likelihoods: Array[Double]) {
+
+  val alt = variant.getAlternateAllele.toString
```
Comment (fnothaft) [inline avocado-core/src/main/scala/org/bdgenomics/avocado/genotyping/annotators/Annotator.scala]: I don't think you need the `.toString` calls.

File: avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala (hunk 1 above)
Comment (fnothaft) [inline avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala]: If you run `./scripts/format-sources` that should do the trick!

Comment (andrewmchen) [inline avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala]: I think that script is only in the ADAM repository. Can move/run the same script?

Comment (fnothaft) [inline avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala]: Hah! Sorry, silly of me... Yes, I'll open a PR later to move it.

**Comment by andrewmchen** [comment]
Added and squashed changes (toString and license)

**Comment by fnothaft** [comment]
Merged! Thanks @andrewmchen!

**Pull request merged**